source/includes/pipeline_config/_015-job.md.erb

```diff
@@ -0,0 +1,30 @@
+<%
+json ='{
+  "jobs": [
+    {
+      "name": "my_job",
+      "run_instance_count": null,
+      "timeout": 0,
+      "environment_variables": [],
+      "resources": [
+        "Linux",
+        "Java"
+      ],
+      "tasks": []
+    }
+  ]
+}'
+%>
+<%=
+describe_sub_object 'The job object', json: json do
+  name                      'String',              'The name of the job.'
+  run_instance_count        'Number/String',       'The number of matching(resources) agent instances on which Job will run.',one_of: %w(all integer)
```

Comment:
I *think* null is also a permitted value, in which case it defaults. Could you please check?